exams/pearson/upload.py

```diff
@@ -28,6 +28,10 @@ def get_connection():
         if getattr(settings, key) is None:
             missing_settings.append(key)
 
+    pearson_exams_sync = settings.FEATURES.get("PEARSON_EXAMS_SYNC", False)
```

Comment:
@amir-qayyum-khan what conditions are you seeing `get_connection()` returning `None`?